test/registerTest.js

```diff
@@ -110,6 +120,30 @@ describe('register', function() {
 		});
 	});
 
+	describe('should output metrics as Protobuf', function() {
+		// If length delimiting is incorrect, it could work for one metric and
+		// 	 not multiple, or vise versa - Test both
+		var metric;
+		beforeEach(function() {
```

Comment:
will do. i was thinking maybe the top level `afterEach` took care of this, but wasn't sure.